android/app/src/main/res/values/strings.xml

```diff
@@ -1,6 +1,7 @@
 <resources>
     <string name="app_name" translatable="false">Open Event</string>
 
+    <string name="url">2017.fossasia.org</string>
```

Comment:
@championswimmer yes you are right, the generator has to be modified to put the event url inside `config.json` so ideally this should be fetched from there.

@arp95 closing this for now as placing this inside `strings.xml` isn't practical as such.

I'll ping you up once this is implemented inside generator.